control-plane/aegisops_control_plane/phase29_mlflow_shadow_model_registry.py

```
@@ -0,0 +1,472 @@
+        return experiment.experiment_id
+    return client.create_experiment(experiment_name, tags=experiment_tags)
+
+
+def _ensure_registered_model(
+    *,
+    client: Phase29MlflowClient,
+    registered_model_name: str,
+) -> None:
+    try:
+        existing = client.get_registered_model(registered_model_name)
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)